counterparty-core/counterpartycore/cli.py

```diff
@@ -33,9 +33,9 @@ def float_range_checker(arg):
         ("-v", "--verbose"),
         {
             "dest": "verbose",
-            "action": "store_true",
-            "default": False,
-            "help": "sets log level to DEBUG",
+            "action": "count",
+            "default": 0,
+            "help": "verbose output (repeat for increased verbosity)",
```

Comment:
Let's specify that `-v` means `DEBUG` and `-vv` means `TRACE`... otherwise someone might try `-vvv` :)

Reply:
done